@here/harp-mapview-decoder/test/TileLoaderTest.ts

```diff
@@ -189,23 +188,31 @@ describe("TileLoader", function() {
                 new MockTileDecoder(),
                 0
             );
+            // This test writes an error to the console, so we disable it.
```

Comment:
It would be better to disable and enable logging with the test setup hooks(`before`, `after`). If the test fails, i.e., the `loadPromise` isn't rejected the logging is not enabled. This could have a side effect on other tests.
Something like:
```javascript
before(function () {
    LoggerManager.instance.update("TileLoader", { enabled: false });
}
after(function () {
    LoggerManager.instance.update("TileLoader", { enabled: true });
}
```

Reply:
then you disable it for all tests, but I want to disable it just for these specific tests.